java/dev/checkstyle/checkstyle.xml

```diff
@@ -218,14 +219,8 @@
         <module name="JavadocMethod">
             <property name="scope" value="public"/>
             <property name="allowMissingParamTags" value="true"/>
-            <property name="allowMissingThrowsTags" value="true"/>
```

Comment:
Can you explain the removal of these rules? Are they no longer available and is there an alternative?